docs/docs-beta/docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Dagster quickstart"
+description: Learn how to quickly get up and running with Dagster
+sidebar_position: 30
+sidebar_label: "Quickstart"
+---
+
+# Build your first Dagster project
+
+Welcome to Dagster! In this guide, you'll use Dagster to create a basic Extract, Transform, Load (ETL) pipeline that extracts data from a CSV file and transforms it.
+
+## What you'll learn
+
+- How to set up a basic Dagster project
+- How to create a Dagster asset for each step of the ETL process
+- How to use Dagster's built-in features to monitor and execute your pipeline
+
+## Prerequisites
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- Basic Python knowledge
+- Python 3.8+ installed on your system. Refer to the [Installation guide](/getting-started/installation) for information.
+</details>
+
+## Step 1: Set up the Dagster environment
+
+1. Open the terminal and create a new directory for your project:
+
+   ```bash
+   mkdir dagster-quickstart
+   cd dagster-quickstart
+   ```
+
+2. Create and activate a virtual environment:
+
+   <Tabs>
+   <TabItem value="macos" label="MacOS">
+   ```bash
+   python -m venv venv
+   source venv/bin/activate
+   ```
+   </TabItem>
+   <TabItem value="windows" label="Windows">
+   ```bash
+   python -m venv venv
+   source venv\Scripts\activate
+   ```
+   </TabItem>
+   </Tabs>
```

Comment:
We'll likely want to do this OS-specific tabs in the installation doc too

Reply:
Oh good call, will do